mex/model/entities/bibliographic-resource.json

```
@@ -0,0 +1,510 @@
+                        "B."
+                    ],
+                    "type": "string"
+                },
+                {
+                    "type": "null"
+                }
+            ],
+            "default": null,
+            "description": "The name of the chapter of the publication, the book section belongs to."
+        },
+        "subtitle": {
+            "default": [],
+            "description": "The subtitle of the publication.",
+            "items": {
+                "$ref": "/schema/fields/text"
+            },
+            "subPropertyOf": [
+                "http://purl.org/dc/terms/title"
+            ],
+            "type": "array"
+        },
+        "title": {
+            "description": "The full title of the publication.",
+            "items": {
+                "$ref": "/schema/fields/text"
+            },
+            "minItems": 1,
+            "sameAs": [
+                "http://purl.org/dc/terms/title"
+            ],
+            "type": "array"
+        },
+        "titleOfBook": {
+            "default": [],
+            "description": "The title of the book in which the book section is published.",
+            "items": {
+                "$ref": "/schema/fields/text"
+            },
+            "subPropertyOf": [
+                "http://purl.org/dc/terms/title"
+            ],
+            "type": "array"
+        },
+        "titleOfSeries": {
+            "default": [],
+            "description": "The title of the book series, the book belongs to.",
+            "items": {
+                "$ref": "/schema/fields/text"
+            },
+            "subPropertyOf": [
+                "http://purl.org/dc/terms/title"
+            ],
+            "type": "array"
+        },
+        "volume": {
+            "anyOf": [
+                {
+                    "examples": [
+                        "1",
+                        "2",
+                        "12"
+                    ],
+                    "type": "integer"
+                },
+                {
+                    "examples": [
+                        "Q3",
+                        "11/12",
+                        "Winter '23"
+                    ],
+                    "type": "string"
+                },
+                {
+                    "type": "null"
+                }
+            ],
+            "default": null,
+            "description": "The volume of the periodical."
+        },
+        "volumeOfSeries": {
+            "anyOf": [
+                {
+                    "examples": [
+                        "1",
+                        "2",
+                        "12"
+                    ],
+                    "type": "integer"
+                },
+                {
+                    "examples": [
+                        "Q3",
+                        "11/12",
+                        "Winter '23"
+                    ],
+                    "type": "string"
+                },
```

this might be quite unstable, i'd rather just tread everything as a string (or null) then

```suggestion
                {
                    "examples": [
                        "2",
                        "Q3",
                        "11/12",
                        "Winter '23"
                    ],
                    "type": "string"
                },
```